app/views/events/index.html.erb

```diff
@@ -0,0 +1,12 @@
+ <h1 class="title mt-5">Comming Events</h1>
+   <div class="columns">
+  <% Event.comming_events.each do |ce| %>
+    <%= render partial: 'events/event_card', :locals => {:ce => ce} %> 
```

Comment:
`Event.coming_events` should be encapsulated to the EventsController's index action as e.g. `@coming_events = Event.coming_events `. It is cleaner to put such logic into the controllers rather than the views.